website/pages/partials/jwt_claim_mapping_details.mdx

```diff
@@ -0,0 +1,71 @@
+## Trusted Identity Attributes via Claim Mappings
+
+Data from JWT claims can be returned from the authentication step as trusted
+identity attributes for use in binding rule selectors and bind name
+interpolation.
+
+Control of which claims are mapped to which identity attributes is governed by
+the [`ClaimMappings`](#claimmappings) and
+[`ListClaimMappings`](#listclaimmappings). These are both maps of items to copy
+with elements of the form: `"<JWT claim>":"<attribute suffix>"`.
+
+The only difference between these two types of mappings is that `ClaimMappings`
+is used to map values which can be interpolated while `ListClaimMappings` is
```

Comment:
`values which can be interpolated`

I'm not sure what that means. Is it simpler to say "scalar values" or "singular values"? If those are not accurate and this is maybe consider giving examples in parentheses?

Reply:
Clarified.